docs/converters/tabular_converter.md

```diff
@@ -300,3 +300,36 @@ Then the following IDs will be generated / retrieved:
   `{"table": "Transformer loads", "name": "load", "key" {"Node_Number": 103, "Subnumber": 1} -> 7`
 * `sym_load.node`:
   `{"table": "Transformer loads", "name": "internal_node", "key" {"Node_Number": 103, "Subnumber": 1} -> 6`
+
+## Secure Handling
+Mapping files enable the specification of custom mappings or filter functions. These functions can come from the `power-grid-model-io` library, be user-provided, or even supplied by third parties. To ensure security, we have implemented several measures and recommend best practices to prevent malicious code execution.
```

Comment:
```suggestion
Mapping files enable the specification of custom mappings or filter functions. These functions can come from the `power-grid-model-io` library, be user-provided, or even supplied by third parties. To ensure security, we have implemented several measures and recommended best-practices to prevent malicious code execution.
```

Reply:
hmmm, We implemented measures before, we recommend practice now in the document....

Reply:
In that case better break up the sentence in two. 
```suggestion
Mapping files enable the specification of custom mappings or filter functions. These functions can come from the `power-grid-model-io` library, be user-provided, or even supplied by third parties. To ensure security, we implemented several measures. Best practices are recommended to prevent malicious code execution.
```